src/kde_analysis/eval_m1m2chieff_poiss_weights.py

```diff
@@ -41,7 +40,8 @@
 parser.add_argument('--discard', default=100, type=int, help='discard first DISCARD iterations')
 parser.add_argument('--start-iter', type=int, help='start at iteration START_ITER after discards')
 parser.add_argument('--end-iter', type=int, help='end at iteration END_ITER after discards')
-
+parser.add_argument('--integrate-kde', type=str, default='marginalized', choices=['marginalized', 'numeric'], help='KDE integration method: "marginalized" (integrate analytically) or "numeric" (3D KDE then numerical integration)')
+#  Marginalization in any dimension
```

Comment:
```suggestion
parser.add_argument('--integrate-kde', type=str, default='marginalized', choices=['marginalized', 'numeric'], help='KDE integration method: "marginalized" (integrate analytically) or "numeric" (3D KDE then numerical integration)')
```
leftover comment line ??

Reply:
removed